test/integration/update.js

```diff
@@ -81,6 +81,45 @@ describe('Update file', () => {
     })
   })
 
+  describe('executable change', () => {
+    it('should ignore them', async () => {
+      const file = await builders.remote.file()
+        .named('file')
+        .data('Initial content')
+        .executable(false)
+        .create()
+      await helpers.remote.pullChanges()
+      await helpers.syncAll()
+      let doc = await pouch.byRemoteIdMaybeAsync(file._id)
+      should(doc).not.have.property('executable')
+
+      await cozy.files.updateAttributesById(file._id, {executable: true})
+      await helpers.remote.pullChanges()
+      await helpers.syncAll()
+
+      doc = await pouch.byRemoteIdMaybeAsync(file._id)
+      should(doc.errors).be.undefined()
+      should(doc).have.property('executable', true)
+
+      const lastRev = doc._rev
+
+      // up-to-date local event
+      await prep.updateFileAsync('local', _.defaults(
+        {
+          executable: process.platform === 'win32' ? 'na' : true
```

Comment:
This makes no sense with the new fix. Let's just drop it.

Reply:
Do you mean dropping the whole test, or just this line?

Reply:
I dropped the whole test.